src/GafferUI/GraphGadget.cpp

```diff
@@ -933,15 +933,19 @@ bool GraphGadget::buttonPress( GadgetPtr gadget, const ButtonEvent &event )
 		{
 			Gaffer::Node *node = nodeGadget->node();
 			bool shiftHeld = event.modifiers && ButtonEvent::Shift;
+			bool controlHeld = event.modifiers && ButtonEvent::Control;
```

Comment:
There's a subtle bug here I think - we need to use a bitwise and (`&`) rather than a logical and (`&&`), otherwise `controlHeld` is true any time _any_ modifier is set. We appear to have the same bug already existing in the line above, but not at line 948 where we were originally testing for the control key. Could you fix the problem on this line and the line above?